File: StudyDemo/app/src/main/java/com/phs1024/studydemo/theory/MainActivity.java
package com.phs1024.studydemo.theory;

import androidx.appcompat.app.AppCompatActivity;

import android.graphics.Color;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.TextView;

import com.phs1024.studydemo.R;

/**
 * @author PHS1024
 */
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView tvHello = findViewById(R.id.tv_hello);
        ImageView ivMain = findViewById(R.id.iv_main);
        ivMain.setImageResource(R.drawable.ic_launcher_background);


    }
}

File: AnimationDemo/app/src/main/java/com/minorpeng/animationdemo/MainActivity.kt
package com.minorpeng.animationdemo

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.animation.AnimationUtils
import android.view.animation.TranslateAnimation
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // val translateAnimation = TranslateAnimation(0f, 500f, 0f, 200f)
        // translateAnimation.duration = 3000
        val translateAnimation = AnimationUtils.loadAnimation(this, R.anim.tranlate_test)
        tv_hello.startAnimation(translateAnimation)

        val scaleAnimation = AnimationUtils.loadAnimation(this, R.anim.scale_test)
        // tv_hello.startAnimation(scaleAnimation)

    }
}
